README.md

```diff
@@ -33,54 +34,54 @@ By default, the prefix is `/lambda/mipsSecret`, resulting the following required
-optionally deduplicate codes, removing inactive codes and `CodesToOmit`.
-Defining any non-false value for this parameter will enable it.
+An `enable_other_code` parameter is available to optionally include an "Other"
+entry in the output with a value from the `OtherCode` parameter. Defining any
+non-false value for this parameter will enable it.
 
-A `disable_inactive_filter` parameter is available to modify the behavior of
-`enable_code_filter`. If this parameter is set to any non-false value, inactive
-codes are no longer removed. Codes are still deduplicated and `CodesToOmit` is
-respected. This parameter has no effect if `enable_code_filter` is not enabled.
+A `disable_no_program_code` parameter is available to optionally exclude a "No
+Program" entry from the output, otherwise it will be injected with a value from
+the `NoProgramCode` parameter. Defining any non-false value for this parameter
+will enable it.
 
-An `enable_other_code` parameter is available for either endpoint to optionally
-include an "Other" entry in the output with a value from the `OtherCode`
-parameter. Defining any non-false value for this parameter will enable it.
+A `disable_inactive_filter` parameter is available to modify how inactive codes
+are treated. If this parameter is set to any non-false value, inactive codes are
+no longer removed. Codes are still deduplicated and `CodesToOmit` is respected.
 
-A `disable_no_program_code` parameter is available for either endpoint to optionally
-exclude a "No Program" entry from the output, otherwise it will be injected
-with a value from the `NoProgramCode` parameter.
-Defining any non-false value for this parameter will enable it.
+An `only_codes` parameter is available to remove all entries except those listed
+here, or as configured in `enable_other_code` and `disable_no_program_code`; any
+`limit` is still applied.
 
-A `limit` parameter is available for either endpoint to restrict the number of
-items returned. This value must be a positive integer, a value of zero
-disables the parameter.
+A `limit` parameter is available to restrict the number of items returned. This
+value must be a positive integer, a value of zero disables the parameter.
 
-| Query String Parameter | Valid Routes | Default Value |
-| --- | --- | --- |
-| enable\_code\_filter | /accounts | Undefined (disabled) |
-| enable\_other\_code | /accounts /tags | Undefined (disabled) |
-| disable\_no\_program\_code | /accounts /tags | Undefined (disabled) |
-| limit | /accounts /tags | `0` (disabled) |
+| Query String Parameter  | Allowed Values                        |
+| ----------------------- | ------------------------------------- |
+| enable_other_code       | "on" or "yes" or "true"               |
+| disable_no_program_code | "on" or "yes" or "true"               |
+| disable_inactive_filter | "on" or "yes" or "true"               |
+| only_codes              | Comma-seperated list of numeric codes |
```

Comment:
are `numeric codes` just the numbers ( i.e. "99030000") or a number and description ( i.e. "99030000": "Platform Infrastructure")?  Could you please clarify the docs?

Reply:
yes, by `numeric codes` I mean just the numbers

Reply:
could you please provide an explicit explanation in the docs?

Reply:
I've added an example value to the description